test/fc_blas/src/CMakeLists.txt

```diff
@@ -1,12 +1,31 @@
+if(CMAKE_Fortran_COMPILER_ID MATCHES PGI)
+    # remove -rdynamic flag offensive for PGI Fortran
+    set(CMAKE_SHARED_LIBRARY_LINK_Fortran_FLAGS)
+endif()
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES Intel AND ENABLE_STATIC_LINKING)
+    # prevent "ifort: ... warning #10121: overriding '-static-intel' with
+    # '-i_dynamic'"
+    SET(CMAKE_SHARED_LIBRARY_LINK_Fortran_FLAGS)
+endif()
+
 if(BLAS_FOUND)
-    if(CMAKE_Fortran_COMPILER_ID MATCHES PGI)
```

Comment:
Who know what adaptations will come in for ACML, ESSL libraries, or for the IBM XL compiler ?